shared/services/rocketpool/client.go

```diff
@@ -784,9 +783,10 @@ func (c *Client) RunPruneProvisioner(container string, volume string, image stri
 
 }
 
-// Runs the prune provisioner
-func (c *Client) RunNethermindPruneStarter(container string) error {
-	cmd := fmt.Sprintf("docker exec %s %s %s", container, nethermindPruneStarterCommand, nethermindAdminUrl)
+// Executes a Go program that triggers NM pruning
+func (c *Client) RunNethermindPruneStarter() error {
+	cmd := fmt.Sprintf(`docker run --rm --network container:rocketpool_eth1 rocketpool/nm-prune-starter %s`, nethermindAdminUrl)
```

Comment:
first, time to make this into a function: https://github.com/rocket-pool/smartnode/blob/d30dd1ff2097c5430d82492ab2199d4f41c7fa7e/rocketpool-cli/service/service.go#L1028

second,

```suggestion
	cmd := fmt.Sprintf(`docker run --rm --network container:%s rocketpool/nm-prune-starter %s`, newFunction(), nethermindAdminUrl)
```

Reply:
one more thing: consider adding a `--name` flag to the container so it is easy to identify if it gets stuck or hangs indefinitely or whatever.

something like `--name %s` which is populated respecting the project name prefix and some constant container name, like `nm-prune-starter`